src/Api/Billing/Models/Responses/PaymentInformationResponse.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Bit.Core.Enums;
+using Bit.Core.Models.Business;
+
+namespace Bit.Api.Billing.Models.Responses;
+
+public record PaymentInformationResponse(PaymentMethod PaymentMethod, TaxInformation TaxInformation)
+{
+    public static PaymentInformationResponse From(BillingInfo.BillingSource billingSource, TaxInfo taxInfo)
```

Comment:
Inline with my comment [here](https://github.com/bitwarden/server/pull/4093/files#r1603869906), this somewhat defeats the purpose of the `ProviderPaymentInfoDTO`. 

I think what we'd want is:
Service Layer: Returns a `ProviderPaymentInforDTO`
API Layer: `PaymentInformationResponse.From(ProviderPaymentInfoDTO)`, if that makes sense.